### PR TITLE
[libphonenumber] Update libphonenumber to 8.12.33. Fixes JB#55699

### PR DESCRIPTION
**Pull request by okodron**
Drop already merged 0001-Add-ability-for-the-C-library-to-link-against-protob.patch.

### CODE REVIEW

File: rpm/libphonenumber.spec [hunk 1]
```diff
@@ -1,26 +1,21 @@
 Name:           libphonenumber
 Summary:        A library for manipulating international phone numbers
-Version:        8.12.12
+Version:        8.12.33
 Release:        1
 License:        ASL 2.0 and BSD and MIT
-URL:            https://github.com/googlei18n/libphonenumber/
+URL:            https://src.fedoraproject.org/rpms/libphonenumber/
```
Comment (pvuorela) [inline rpm/libphonenumber.spec]: Could assume google thing being a better upstream url?

**Comment by pvuorela** [comment]
Having some problem building locally but OBS went fine. Anyhow wouldn't expect related to these changes. Overall looks good to me.

File: rpm/libphonenumber.spec (hunk 1 above)
Comment (okodron) [reply rpm/libphonenumber.spec]: Thanks - accidental mistake.

**Comment by pvuorela** [comment]
LGTM

**Pull request merged**